bin/sync/article_updater.rb

```
@@ -0,0 +1,149 @@
+    draft: false
+    tags:
+    #{article_data['tags'].map { |tag| "  - #{tag}" }.join("\n")}
+    canonical_url: "#{article_data['canonical_url']}"
+    cover_image: "#{cover_image}"
+    slug: "#{slug}"
+    #{metatags_image.strip}
+    ---
+    #{article_data['body_markdown']}
+    MARKDOWN
+  end
+end
```

rubocop

[convention] Module has too many lines. [115/100]

(Metrics/ModuleLength)